<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Client</class>
 <widget class="QWidget" name="Client">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Client</string>
  </property>
  <widget class="QLineEdit" name="lEdit_ip">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>30</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>192.56.12.3</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lEdit_port">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>1234</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pBtn_connect">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>210</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Conectar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pBtn_close">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>220</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Cerrar</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_message">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>160</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Mensaje recibido</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
